frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -125,7 +156,9 @@ export const funnelLogic = kea<funnelLogicType<TimeStepOption>>({
                             ? { breakdown: null, breakdown_type: null }
                             : {}),
                     }
-                    let result: FunnelResult
+
+                    let result
```

Comment:
Tricky: `pollFunnel` would ideally return a `FunnelResult` or `FunnelResultWithBreakdown` based on whether its input params include `breakdown`. But I don't want to handle the possibility of `FunnelResultWithBreakdown` and `FunnelResult` everywhere, which is why the `aggregateBreakdownResult` function comes in to reshape the data.